raster.py

```diff
@@ -317,5 +371,113 @@ def cellulose_isoscape_geotiff() -> AmazonGeoTiff:
+  global krig_variances_isoscape_geotiff_
+  if not krig_variances_isoscape_geotiff_:
+    krig_variances_isoscape_geotiff_ = load_named_raster(get_raster_path("uc_davis_d18O_cel_ordinary_random_grouped_vars.tiff"), "ordinary_krig_vars")
+  return krig_variances_isoscape_geotiff_  
+
+precipitation_regression_isoscape_geotiff_ = None
+def precipitation_regression_isoscape_geotiff() -> AmazonGeoTiff:
+  global precipitation_regression_isoscape_geotiff_
+  if not precipitation_regression_isoscape_geotiff_:
+    precipitation_regression_isoscape_geotiff_ = load_named_raster(get_raster_path("isoscape_fullmodel_d18O_prec_REGRESSION.tiff"), "precipitation_regression_isoscape_geotiff")
+  return precipitation_regression_isoscape_geotiff_
+
+craig_gordon_isoscape_geotiff_ = None
+def craig_gordon_isoscape_geotiff() -> AmazonGeoTiff:
+  global craig_gordon_isoscape_geotiff_
+  if not craig_gordon_isoscape_geotiff_:
+    craig_gordon_isoscape_geotiff_ = load_named_raster(get_raster_path("Iso_Oxi_Stack_mean_TERZER.tif"), "craig_gordon_isoscape_geotiff")
+  return craig_gordon_isoscape_geotiff_
+
+brisoscape_geotiff_ = None
+def brisoscape_geotiff() -> AmazonGeoTiff:
+  global brisoscape_geotiff_
+  if not brisoscape_geotiff_:
+    brisoscape_geotiff_ = load_named_raster(get_raster_path("brisoscape_mean_ISORIX.tif"), "brisoscape_geotiff")
+  return brisoscape_geotiff_
+
+d13C_mean_geotiff_ = None
+def d13C_mean_geotiff() -> AmazonGeoTiff:
+  global d13C_mean_geotiff_
+  if not d13C_mean_geotiff_:
+    d13C_mean_geotiff_ = load_named_raster(get_raster_path("d13C_cel_map_BRAZIL_stack.tiff"), "d13C_mean", use_only_band_index=0)
+  return d13C_mean_geotiff_
+
+d13C_var_geotiff_ = None
+def d13C_var_geotiff() -> AmazonGeoTiff:
+  global d13C_var_geotiff_
+  if not d13C_var_geotiff_:
+    d13C_var_geotiff_ = load_named_raster(get_raster_path("d13C_cel_map_BRAZIL_stack.tiff"), "d13C_var", use_only_band_index=1)
+  return d13C_var_geotiff_
+
+# A collection of column names to functions that load the corresponding geotiffs.
+column_name_to_geotiff_fn = {
+  "VPD" : vapor_pressure_deficit_geotiff,
+  "RH": relative_humidity_geotiff,
+  "PET": pet_geotiff,
+  "DEM": dem_geotiff,
+  "PA": pa_geotiff,
+  "Mean Annual Temperature": temperature_geotiff,
+  "Mean Annual Precipitation": brazil_map_geotiff,
+  "Iso_Oxi_Stack_mean_TERZER": craig_gordon_isoscape_geotiff,
+  "isoscape_fullmodel_d18O_prec_REGRESSION": precipitation_regression_isoscape_geotiff,
+  "brisoscape_mean_ISORIX": brisoscape_geotiff,
+  "d13C_cel_mean": d13C_mean_geotiff,
+  "d13C_cel_var": d13C_var_geotiff,
+  "ordinary_kriging_linear_d18O_predicted_mean" : krig_means_isoscape_geotiff,
+  "ordinary_kriging_linear_d18O_predicted_variance" : krig_variances_isoscape_geotiff
+}
+
+def generate_isoscapes_from_variational_model(
+    output_geotiff_id: str,
+    model: tf.keras.Model,
+    feature_transformer: ColumnTransformer,
+    required_geotiffs: List[str],
```

Comment:
Do we need to specify here that they need to be in the order of the model input.
Also, is there a way to get this from the feature_transformer or some metadata of the model so we don't need to "keep track" of the order of the columns?

Reply:
I couldn't find a to store the column training info as part of the model. In a follow up set of PRs I just train them and load them in alphabetical order.

Reply:
Also, FeatureTransformer does appear to have the columns in order they were trained, but I don't think that's a guarantee. I'd rather be responsible for sorting it ourselves.